ansible/provision.yml

```diff
@@ -27,6 +27,14 @@
     - common
     - mongodb
 
+- name: Message queue
+  user: ubuntu
+  hosts: tag_type_mq:mq
```

Comment:
I'm wrong again.  We should clarify what this is trying to do.

the `:` is an or, we may want to get rid of this and just have `mq`